client/table.go

```diff
@@ -205,15 +205,80 @@ func (m *model) encodeColumnKey(primaryKey []byte, colID uint32) []byte {
 	return roachencoding.EncodeUvarint(key, uint64(colID))
 }
 
+// CreateNamespace creates a new namespace.
+//
+// TODO(pmattis): Is "namespace" the correct terminology? PostgreSQL and MySQL
+// use "database". PostgreSQL also has the notion of a schema. A
+// fully-qualified name in PostgreSQL looks like
+// "<database>.<schema>.<table>". See
+// http://www.postgresql.org/docs/9.4/static/ddl-schemas.html.
```

Comment:
oracle does the same. I think "database" is meant to be the place where you configure resource requirements and such. Schema is the place to describe your problem domain as tables and constraints.